src/platforms/unity/migration.mdx

```diff
@@ -29,6 +30,14 @@ You no longer need to to call `SentryUnity.Init`. Instead, you can let the SDK s
 
 Learn more in our [options documentation](/platforms/unity/configuration/options/).
 
+## Migrating From 0.7.0 or Earlier
+
+Starting from version 0.7.0, the SDK is [aliasing](https://blog.sentry.io/2022/02/24/alias-an-approach-to-net-assembly-conflict-resolution) its dependencies. We do this to avoid creating conflicts with other packages. This also means that if you were relying on our dependencies for certain types, then you will have to import them into your project yourself going forward.
+
```

Comment:
```suggestion
Starting from version 0.7.0, the SDK is [aliasing](https://blog.sentry.io/2022/02/24/alias-an-approach-to-net-assembly-conflict-resolution) its dependencies. We do this to avoid creating conflicts with other packages. This also means that if you were relying on our dependencies for certain types, then you'll have to import them into your project yourself going forward.

```
Could you add a descriptor to "certain types" - certain types of what?

Reply:
Sure thing!